packages/web/src/pages/oauth-login-page/utils.ts

```diff
@@ -127,38 +126,7 @@ export const formOAuthResponse = async ({
     email = userEmail
   }
 
-  const storageNodeSelector = await getStorageNodeSelector()
-  let profilePicture:
-    | { '150x150': string; '480x480': string; '1000x1000': string }
-    | undefined
-  if (account.profile_picture_sizes) {
-    const storageNode = storageNodeSelector.getNodes(
-      account.profile_picture_sizes
-    )[0]
-    const base = `${storageNode}/content/`
-    profilePicture = {
-      '150x150': `${base}${account.profile_picture_sizes}/150x150.jpg`,
-      '480x480': `${base}${account.profile_picture_sizes}/480x480.jpg`,
-      '1000x1000': `${base}${account.profile_picture_sizes}/1000x1000.jpg`
-    }
-    if (account.profile_picture_cids) {
-      if (account.profile_picture_cids[SquareSizes.SIZE_150_BY_150]) {
-        profilePicture['150x150'] = `${base}${
-          account.profile_picture_cids[SquareSizes.SIZE_150_BY_150]
-        }`
-      }
-      if (account.profile_picture_cids[SquareSizes.SIZE_480_BY_480]) {
-        profilePicture['480x480'] = `${base}${
-          account.profile_picture_cids[SquareSizes.SIZE_480_BY_480]
-        }`
-      }
-      if (account.profile_picture_cids[SquareSizes.SIZE_1000_BY_1000]) {
-        profilePicture['1000x1000'] = `${base}${
-          account.profile_picture_cids[SquareSizes.SIZE_1000_BY_1000]
-        }`
-      }
-    }
-  }
+  const profilePicture = account.profile_picture
```

Comment:
this should be
account.profile_picture_sizes ?? account.profile_picture 

 i think

Reply:
Oh dang, we haven't cleaned this up yet?

Reply:
we actually should have in almost all places. anywhere that uses the artwork field is good and doesn't need to worry about this CID at all